app/models/setting.js

```diff
@@ -9,7 +9,7 @@ export default Model.extend(ValidationEngine, {
     title: attr('string'),
     description: attr('string'),
     logo: attr('string'),
-    cover: attr('string'),
+    cover_image: attr('string'),
```

Comment:
I will try to make it consistent now, if i can't make it today i will raise an issue for later.